docs/scripts/buildApi.ts

```diff
@@ -339,8 +387,8 @@ async function buildDocs(options: {
   };
 
   // styled components does not have the options static
-  const styledComponent = !component?.default?.options;
-  if (styledComponent) {
+  const nonJSSComponent = !component?.default?.options;
```

Comment:
This name better reflects the condition as it is true for both components generated with the styled utility, as well as the unstyled components